RELEASE-NOTES.txt

```diff
@@ -2,6 +2,9 @@
 
 9.3
 -----
+- [*] Support Arabic numerals on amount fields. [https://github.com/woocommerce/woocommerce-ios/pull/6891]
```

Comment:
nit: this might not be intentional?

Reply:
Good catch! It duplicates another row now. Apparently wrong conflict resolution somewhere happened

I fixed it here 87124a51a9948b6342621b8dc3695a5e2999cd69